ui/components/component-library/tag/tag.scss

```diff
@@ -0,0 +1,6 @@
+.tag {
+  height: auto;
+  width: max-content;
+  padding-top: 2px;
+  padding-bottom: 2px;
+}
```

Comment:
You will also need to add `border-radius: 9999px;` until this PR has been merged: https://github.com/MetaMask/metamask-extension/pull/16118

Reply:
Updated